src/ResourceManager/Aks/Commands.Aks/help/Stop-AzureRmAksDashboard.md

```diff
@@ -45,9 +45,32 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -PassThru
+{{Fill PassThru Description}}
```

Comment:
@maddieclayton the description for `PassThru` is missing in a few markdown files